server/aws-lsp-codewhisperer/src/language-server/localProjectContext/initializeUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { InitializeParams, WorkspaceFolder } from '@aws/language-server-runtimes/protocol'
+import * as path from 'path'
+import { URI } from 'vscode-uri'
+
+export function getWorkspaceFolders(params?: InitializeParams): WorkspaceFolder[] {
```

Comment:
Should this method be wrapped in a try catch, returning an empty array if there is an error? In cases like VS, the inputs here would be out of the extension and user's control but we may not want the server to crash (or fail to initialize).

Reply:
Updated